### PR TITLE
Config report following both the failed and successful job runs.

### PR DESCRIPTION
**Pull request by amolpati30**
Ansible tasks which make changes on a host are displayed as notices in the config report, and failed Ansible tasks are displayed as errors in the config report

Dependent PR: https://github.com/SatelliteQE/airgun/pull/1479

### CODE REVIEW

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7872
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 42 warnings in 1275.57s (0:21:15) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7903
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 49 warnings in 2099.65s (0:34:59) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7964
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 45 warnings in 1297.77s (0:21:37) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7969
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 45 warnings in 1078.32s (0:17:58) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7970
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 46 warnings in 1099.29s (0:18:19) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7971
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 45 warnings in 1097.49s (0:18:17) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7979
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 26 deselected, 45 warnings in 1022.86s (0:17:02) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8016
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : =========== 26 deselected, 32 warnings, 1 error in 202.77s (0:03:22) ===========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8024
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : =========== 26 deselected, 32 warnings, 1 error in 203.81s (0:03:23) ===========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8045
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 17 deselected, 35 warnings in 1149.28s (0:19:09) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8072
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 17 deselected, 36 warnings in 1184.09s (0:19:44) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8075
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 17 deselected, 36 warnings in 974.61s (0:16:14) ===========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

File: tests/foreman/ui/test_ansible.py [hunk 1]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+                'name': module_target_sat.hostname,
+                'location-ids': module_location.id,
+                'organization-ids': module_org.id,
+            }
+        )
+        SELECTED_ROLE = 'theforeman.foreman_scap_client'
+        id = module_target_sat.nailgun_smart_proxy.id
+        module_target_sat.api.AnsibleRoles().sync(
+            data={'proxy_id': id, 'role_names': SELECTED_ROLE}
+        )
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:594]:
```suggestion
            SELECTED_ROLE = 'theforeman.foreman_scap_client'
            nc = module_target_sat.nailgun_smart_proxy
            nc.location = [module_location]
            nc.organization = [module_org]
             nc.update(['organization', 'location'])
            module_target_sat.api.AnsibleRoles().sync(data={'proxy_id': nc.id, 'role_names': SELECTED_ROLE})
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8084
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 passed, 17 deselected, 38 warnings in 1225.69s (0:20:25) ==========
```

File: tests/foreman/ui/test_ansible.py [hunk 2]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
+            # check err log for config report after ansible job scheduled
+            session.host_new.run_job(rhel_contenthost.hostname)
+            session.jobinvocation.wait_job_invocation_state(
+                entity_name='Run ansible roles',
+                host_name=rhel_contenthost.hostname,
+                expected_state='failed',
+            )
+            err_log = session.configreport.search(rhel_contenthost.hostname)
+            ROLE = SELECTED_ROLE.split('.')[1]
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:615]: foreman_scap_client is not role, its a package name
```suggestion
            package_name = SELECTED_ROLE.split('.')[1]
```

File: tests/foreman/ui/test_ansible.py [hunk 3]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+                force=True,
+            )
+            assert result.status == 0, f'Failed to register host: {result.stderr}'
+            session.host_new.run_job(rhel_contenthost.hostname)
+            session.jobinvocation.wait_job_invocation_state(
+                entity_name='Run ansible roles', host_name=rhel_contenthost.hostname
+            )
+            notice_log = session.configreport.search(rhel_contenthost.hostname)
+            assert f'notice Install the {ROLE} package' in notice_log['permission_denied']
+            assert 'Installed: rubygem-foreman_scap_client' in notice_log['permission_denied']
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:638]:
```suggestion
            assert f'notice Install the {package_name} package' in notice_log['permission_denied']
            assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
```

File: tests/foreman/ui/test_ansible.py [hunk 4]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            )
+            err_log = session.configreport.search(rhel_contenthost.hostname)
+            ROLE = SELECTED_ROLE.split('.')[1]
+            assert f'err Install the {ROLE} package' in err_log['permission_denied']
+            assert (
+                'Execution error: Failed to install some of the specified packages'
+                in err_log['permission_denied']
+            )
+
+            # check notice log for config report after ansible job scheduled
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:622]:
```suggestion
            # Verify notice logs in config report after ansible role is successfully executed
```

File: tests/foreman/ui/test_ansible.py [hunk 5]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            ansible_roles_table = session.host_new.get_ansible_roles(rhel_contenthost.hostname)
+            assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
+            # check err log for config report after ansible job scheduled
+            session.host_new.run_job(rhel_contenthost.hostname)
+            session.jobinvocation.wait_job_invocation_state(
+                entity_name='Run ansible roles',
+                host_name=rhel_contenthost.hostname,
+                expected_state='failed',
+            )
+            err_log = session.configreport.search(rhel_contenthost.hostname)
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:614]: I'd suggest to add similar comment here about what we're verifying

File: tests/foreman/ui/test_ansible.py [hunk 6]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list('8')
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:560]:
```suggestion
    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

File: tests/foreman/ui/test_ansible.py [hunk 7]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            err_log = session.configreport.search(rhel_contenthost.hostname)
+            ROLE = SELECTED_ROLE.split('.')[1]
+            assert f'err Install the {ROLE} package' in err_log['permission_denied']
+            assert (
+                'Execution error: Failed to install some of the specified packages'
+                in err_log['permission_denied']
+            )
+
+            # check notice log for config report after ansible job scheduled
+            rhel_contenthost.create_custom_repos(client_repo=settings.repos.SATCLIENT_REPO['rhel8'])
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:623]:
```suggestion
            rhel_var = settings.content_host.default_rhel_version
            rhel_contenthost.create_custom_repos(client_repo=settings.repos.SATCLIENT_REPO[f'rhel{rhel_ver}'])
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8089
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 17 deselected, 36 warnings in 1067.04s (0:17:47) ==========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8090
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 failed, 17 deselected, 36 warnings in 1021.17s (0:17:01) ==========
```

File: tests/foreman/ui/test_ansible.py [hunk 8]
```diff
@@ -556,6 +556,85 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            module_org, module_location, module_activation_key.name, module_target_sat
+        )
+        assert result.status == 0, f'Failed to register host: {result.stderr}'
+        with module_target_sat.ui_session() as session:
+            session.location.select(module_location.name)
+            session.organization.select(module_org.name)
+            session.host_new.add_single_ansible_role(rhel_contenthost.hostname)
+            ansible_roles_table = session.host_new.get_ansible_roles(rhel_contenthost.hostname)
+            assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
+            # Verify err log for config report after ansible role is successfully executed
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:607]:
```suggestion
            # Verify error log for config report after ansible role is executed
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8092
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : =========== 17 deselected, 23 warnings, 1 error in 202.00s (0:03:22) ===========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8095
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : =========== 17 deselected, 23 warnings, 1 error in 196.70s (0:03:16) ===========
```

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8100
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 passed, 17 deselected, 38 warnings in 2357.90s (0:39:17) ==========
```

File: tests/foreman/ui/test_ansible.py (hunk 1 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Added `nailgun_smart_proxy` and removed suggested one

File: tests/foreman/ui/test_ansible.py (hunk 2 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: updated object name

File: tests/foreman/ui/test_ansible.py (hunk 3 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Changes has been done here as well.

File: tests/foreman/ui/test_ansible.py (hunk 4 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py (hunk 6 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Added the default RHEL version and removed hardcoded values.

File: tests/foreman/ui/test_ansible.py (hunk 7 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py (hunk 8 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py [hunk 9]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+        """
+        SELECTED_ROLE = 'theforeman.foreman_scap_client'
+        nc = module_target_sat.nailgun_smart_proxy
+        nc.location = [module_location]
+        nc.organization = [module_org]
+        nc.update(['organization', 'location'])
+        id = module_target_sat.nailgun_smart_proxy.id
+        module_target_sat.api.AnsibleRoles().sync(
+            data={'proxy_id': id, 'role_names': SELECTED_ROLE}
+        )
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:591]:
```suggestion
        module_target_sat.api.AnsibleRoles().sync(
            data={'proxy_id': nc.id, 'role_names': SELECTED_ROLE}
        )
```

File: tests/foreman/ui/test_ansible.py [hunk 10]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+        :id: 286048f8-0f4f-4a3c-b5c7-fe9c7af8a780
+
+        :steps:
+            1. Import Ansible Roles
+            2. Assign Ansible roles to a host
+            3. Run Ansible Roles on a host
+            4. Check Config Report
+
+        :expectedresults: Verify that any tasks that make changes on the host are listed as notice in the config reportCaseLevel and
+            Verify that any task failures are listed as errors in the config reportCaseLevel
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:581]:
```suggestion
        :expectedresults: 
                   1. Verify that any tasks that make changes on the host are listed as notice in the config report 
                   2. Verify that any task failures are listed as errors in the config report
```

File: tests/foreman/ui/test_ansible.py [hunk 11]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+    ):
+        """Check that Ansible tasks that make changes on a host show as notice in the config report and
+        failed Ansible tasks show as errors in the config report
+
+        :id: 286048f8-0f4f-4a3c-b5c7-fe9c7af8a780
+
+        :steps:
+            1. Import Ansible Roles
+            2. Assign Ansible roles to a host
+            3. Run Ansible Roles on a host
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:577]:
```suggestion
            2. Assign and Run Ansible roles to a host
```

File: tests/foreman/ui/test_ansible.py [hunk 12]
```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:561]: Could you remove related stubbed test cases as you're automating this test here ?https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_ansible.py#L290-L323

Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py:526]: I have removed these two stubbed tests as they are covered in this test case.

File: tests/foreman/ui/test_ansible.py (hunk 11 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py (hunk 10 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py (hunk 9 above)
Comment (amolpati30) [reply tests/foreman/ui/test_ansible.py]: Done.

File: tests/foreman/ui/test_ansible.py [hunk 13]
```diff
@@ -556,6 +521,87 @@ def test_positive_non_admin_user_access_with_usergroup(
...
+            err_log = session.configreport.search(rhel_contenthost.hostname)
+            package_name = SELECTED_ROLE.split('.')[1]
+            assert f'err Install the {package_name} package' in err_log['permission_denied']
+            assert (
+                'Execution error: Failed to install some of the specified packages'
+                in err_log['permission_denied']
+            )
+
+            # Verify notice log for config report after ansible role is successfully executed
+            rhel_ver = settings.content_host.default_rhel_version
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_ansible.py:585]:
```suggestion
```

**Review by Gauravtalreja1 — COMMENTED** [review]
ACK, overall looks good now, suggesting two minor changes

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
airgun: 1479

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8103
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 passed, 15 deselected, 36 warnings in 1458.04s (0:24:18) ==========
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**